psi4/src/psi4/libfock/cubature.cc

```diff
@@ -2808,14 +2848,16 @@ double NuclearWeightMgr::computeNuclearWeight(MassPoint mp, int A, double stratm
     for (int l = 0; l < natom; l++) dist[l] = distToAtom(mp, l);
 
     double (*stepFunction)(double) = (scheme_ == STRATMANN) ? StratmannStepFunction : BeckeStepFunction;
+    double (*muFunction)(double,double,double) = (scheme_ == SBECKE) ? SmoothBeckeMu : BeckeMu;
```

Comment:
Can you run clang-format over these files?

Reply:
done